catalog/ui/catalog-ui-enumeration/src/main/java/org/codice/ddf/catalog/ui/enumeration/ExperimentalEnumerationExtractor.java

```diff
@@ -40,10 +44,29 @@
 
     private final List<MetacardType> metacardTypes;
 
+    private final List<AttributeInjector> attributeInjectors;
+
+    /**
+     * @deprecated This constructor does not take into account injected attributes.
+     * The other constructor <code>ExperimentalEnumerationExtractor/3</code> should be used.
```

Comment:
✏️ Instead of a fenced code block, you should use `{@link #ExperimentalEnumerationExtractor(AttributeValidatorRegistry, List, List)}`